package.json

```diff
@@ -40,14 +40,14 @@
     "cli-microservice": "sh cli-init.sh"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "2.30.0",
-    "@typescript-eslint/parser": "2.30.0",
-    "eslint": "6.8.0",
+    "@typescript-eslint/eslint-plugin": "3.0.2",
+    "@typescript-eslint/parser": "3.0.2",
+    "eslint": "7.1.0",
     "eslint-config-standard": "14.1.1",
     "eslint-plugin-import": "2.20.2",
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-promise": "4.2.1",
     "eslint-plugin-standard": "4.0.1",
-    "lerna": "3.20.2"
+    "lerna": "3.22.0"
```

Comment:
Please checkout this files and revert the changes

Reply:
@this is older PR plz review new one 
https://github.com/1-Platform/one-platform/pull/407